packages/opentelemetry-core/src/trace/sampler/ProbabilitySampler.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Sampler, SpanContext } from '@opentelemetry/types';
+
+/** Sampler that samples a given fraction of traces. */
+export class ProbabilitySampler implements Sampler {
+  readonly description = 'ProbabilitySampler';
+
+  constructor(readonly probability: number = 1) {}
+
+  shouldSample(parentContext?: SpanContext) {
```

Comment:
This should implement the actual probability so it can be used with values other than `0` and `1`. I think a simple `return Math.random() < this.probability` instead of `return false` would do.

Reply:
Done (I was gonna add it later).